### DISCUSSION TITLE
will ControlNet Models be released?

### DISCUSSION THREAD

**Discussion by TruthSearcher** [PixArt-alpha/PixArt-alpha#34] [General]
Will controlnet models be released and if so what is the timeline?

**Comment by lawrence-cj** [comment]
Since we don't have sufficient data and resource for ControlNet training. We recommend anyone interested in Pixart-alpha to train their own ControlNet model.

**Comment by Njasa2k** [comment]
Does anyone want to use this dataset to make a controlnet model?
[MultiGen-20M](https://console.cloud.google.com/storage/browser/sfr-unicontrol-data-research/dataset;tab=objects?prefix=&forceOnObjectsSortingFiltering=false)

**Comment by betterze** [comment]
@lawrence-cj Could you share the controlnet code? I can train a model by myself. thx a lot for your great work.

**Reply by lawrence-cj** [reply]
Acutally, we realesed the repo style code pxiart-ControlNet here: https://github.com/PixArt-alpha/PixArt-alpha/tree/feat/pixart-ctrlnet-public. We are still working on a diffusers version which may easier to use. ANYWAY, you can try this code out first. We will release the ckpt soon.

**Comment by lawrence-cj** [comment]
ControlNet-related code and ckpts are fully available.

**Reply by Geliv** [reply]
is there a diffusers version of pixart-controlnet? And, can I use pixart-controlnet with custom lora?
